spec/features/activity_pages/activity_pages_spec.rb

```diff
@@ -12,16 +12,40 @@
   context 'as authenticated user' do
     describe 'index page', js: true do
       let(:trackable) { create(:issue) }
+      let(:trackable_card) { create(:card) }
       let(:user) { create(:user) }
-
+      let(:second_user) { create(:user) }
       let(:create_activities) do
-        50.times do
+        35.times do
```

Comment:
Instead of keeping the `X.times` approach, what about using factory bot and calling `create_list`?

Reply:
You're right. It was already there so I didn't really notice. Changed it!